Userland/Libraries/LibWeb/HTML/HTMLIFrameElement.idl

```diff
@@ -16,7 +16,7 @@ interface HTMLIFrameElement : HTMLElement {
     [CEReactions, Reflect] attribute DOMString width;
     [CEReactions, Reflect] attribute DOMString height;
     [FIXME, CEReactions] attribute DOMString referrerPolicy;
-    [FIXME, CEReactions] attribute DOMString loading;
+    [CEReactions, Reflect] attribute DOMString loading;
```

Comment:
This should be limited to known values in the same way as `HTMLImageElement.loading`.